### ISSUE TITLE
vue-datepicker adds 4 hours to the time

### ISSUE THREAD

**Issue by mark1502** [Vuepic/vue-datepicker#396]
This may not be a bug, but simply an question about settings, but if I enter a time with the datepicker, it returns the date and time with 4 hours added to the time.

Here's the html code:
```
<VueDatePicker id="form.date1" name="form.date1" v-model="form.date1 :enable-time-picker="true" :is24="false" />
```
When the date and time are selected, they appear correctly, on the calendar and in the form element.  It also appears correct in the Vue Devtools, though that appears to also indicate east coast time.  

But when the data is sent to the server, the time is 4 hours later.  I assume this has something to do with a timezone setting, but I don't know how to set it.  Basically, I want to record the same time the user enters, regardless of where the user is.

Thanks

**Comment by Jasenkoo** [comment]
It looks like you are converting to UTC when sending it to the server. Take a look at [`utc`](https://vue3datepicker.com/props/modes/#utc) prop.

**Comment by mark1502** [comment]
Thanks for your reply.  I experimented a lot with both the utc and timezone props.  None of them changed the datetime being sent to the server at all, until I tried utc="preserve".  That did what I needed!  

I was a little bit thrown off by the utc prop example, because it looked like I should either add 'utc' as the prop, or not.  Eventually, it occurred to me to try utc="preserve".  Perhaps the example should show utc="true" and maybe a comment to show how to set utc="preserve" if that is desired, for fools like me who didn't realize I could set the prop to "preserve". 

Thanks again for your help and for this excellent datepicker!
